packages/backend/prisma/schema.prisma

```diff
@@ -16,6 +22,17 @@ model accounts {
   skipWaitlist  Boolean       @default(false)
   connections   connections[]
   users         users[]
+  apps          apps[]
+}
+
+model apps {
+  id                         String   @id
+  tp_id                      TP_ID
+  scope                      String[]
+  app_client_id              String?
+  app_client_secret          String?
+  owner_account_public_token String
+  account                    accounts @relation(fields: [owner_account_public_token], references: [public_token])
```

Comment:
Do we need that relation? Ig app should be linked to account as it'll be the same for all connections of a given account and we can get the app for a given connection by fetching the account. Unless, you want to avoid fetching account first in which case yeah we can link app and connection too. Lmk if I need to change it

Reply:
Done